code/drasil-example/Drasil/NoPCM/Requirements.hs

```diff
@@ -39,9 +40,9 @@ findMass = findMassConstruct inputInitVals (phrase mass) [eBalanceOnWtr]
 --
 oIDQVals :: [Sentence]
 oIDQVals = map foldlSent_ [
-  [S "the", plural value, S "from", makeRef2S inputInitVals],
-  [S "the", phrase mass, S "from", makeRef2S findMass],
-  [ch balanceDecayRate, sParen (S "from" +:+ makeRef2S balanceDecayRate)]
+  [pluralNP (the value), fromSource inputInitVals],
+  [phraseNP (the mass), fromSource findMass],
+  [ch balanceDecayRate, fromSource balanceDecayRate]
```

Comment:
Using fromSource on `the value` and `the mass` wraps those references in parenthesis (which was not there previously). Should I keep it this way? The `balanceDecayRate` one already had parenthesis previously, so it worked well.

Reply:
increased uniformity is good, so yes, keep that.